documentation/devonfw-ide-introduction.asciidoc

```diff
@@ -1,12 +1,69 @@
-= Introduction
+= devonfw-ide
 
-http://devonfw.com[devonfw] provides a solution to building applications which combine best-in-class frameworks and libraries
-as well as industry proven practices and code conventions.
-It massively speeds up development, reduces risks and helps deliver better results.
+== What is devonfw-ide?
 
-This document contains the instructions for the tool `devonfw-ide` to set up and maintain your development tools including your favorite IDE (integrated development environment).
+`devonfw-ide` is a toolbox allowing you to pick tools which fit the requirements and needs of your project.
 
-include::features.asciidoc[leveloffset=2]
+It is not another Eclipse. It gives you Eclipse, along with Java, maven, NodeJs, VSCode and everything else you may need to quick start your development. All you need to do is download and extract a zip file and execute its setup, reducing your initial setup time for starting a project from days to minutes.
 
-include::setup.asciidoc[leveloffset=2]
+That is not all. `devonfw-ide` extends the existing shell with helpful commands to automate the provisioning of the development environment.
+
+
+== Why do you need devonfw-ide?
+
+Usually the team members follows wiki/confluence articles at least once to setup the environment. Most of the times, these articles might have outdated some days after writing it. Also there are chances that the team members of a project might have downloaded and using different versions of tools. Setting up the tools manually are tedious and error prone. Following the step by step instructions sometimes throw errors.  Use of different configuration by team members can cause diff-wars. `devon-ide' let you download, install and configure the required tools in a single click.
```

Comment:
Following the step by step instructions sometimes throw errors. -> Better: you sometimes run into errors.

Reply:
"Following the step by step instructions sometimes throw errors. Use of different configuration by team members can cause diff-wars."
This are to different ideas/thoughts. You should separate them more clearly to avoid confusion.